src/main/java/com/oneops/inv/Inventory.java

```diff
@@ -688,9 +689,9 @@ private String computeHostId(CiResource compute) throws InventoryException {
                     JSONObject entries = new JSONObject(entriesStr);
                     ArrayList<String> hostnames = new ArrayList<String>();
                     hostnames.addAll( entries.keySet() );
-                    Collections.sort( hostnames );
-
-                    hostId = hostnames.get(0);
+                    //filter out the ip returned as part of a hostname component with a nasty regex
```

Comment:
Wanna doc what the nasty regex is supposed to remove?

Reply:
Done :)